### ISSUE TITLE
AlignParameters and AlignHash + Class DSLs

### ISSUE THREAD

**Issue by patbenatar** [rubocop/rubocop#687]
In Rails, this kind of thing is quite common to see:

``` ruby
class Model < ActiveRecord::Base
  validates :attribute_1, :attribute_2, :attribute_3, :attribute_4, 
    presence: true
end
```

But you wouldn't as often see the same thing when calling a method with long hash args on an instance.

The `AlignHash` and `AlignParameters` cops allow us to enforce styling like so:

``` ruby
method(
  option_1: true,
  option_2: false
)
```

which I like on the instance level but enforcing on the class DSLs seems to go too much against the Rails conventions for me to want to enforce.

On one side I'm in favor of consistency but on the other I want to stick with the Rails conventions as they are widely adhered to.

Should these cops have an option to behave differently on the class vs instance?
